### ISSUE TITLE
Removing "unique" constraint on column does not actually drop unique

### ISSUE THREAD

**Issue by coufal** [VemtoOrg/vemto2-issues#63]
When removing the `unique` constraint on a column, the migration that gets generated in 2.0.6 does not drop the unique constraint.

Here is an example from Laravel Breeze, where I tried to remove the unique constraint from column `email`:

**Generated migration by Vemto 2.0.6**

```php
public function up()
  {
      Schema::table('users', function (Blueprint $table) {
          $table->string('email', 255)->change();
      });
  }

 public function down()
    {
        Schema::table('users', function (Blueprint $table) {
            $table
                ->string('email', 255)
                ->unique()
                ->change();
        });
    }
```

**Expected migration to actually drop the constraint**

```php
public function up()
    {
        Schema::table('users', function (Blueprint $table) {
            $table->dropUnique('users_email_unique');
        });
    }

public function down()
    {
        Schema::table('users', function (Blueprint $table) {
            $table->unique('email', 'users_email_unique');
        });
    }
```

**Comment by TiagoSilvaPereira** [comment]
@coufal thanks for reporting

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.22
